spec/br_danfe/mdfe_lib/header_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'spec_helper'
+
+describe BrDanfe::MdfeLib::Header do
+  let(:xml_as_string) do
+    <<-eos
```

Comment:
```suggestion
    <<~XML
      <?xml version="1.0" encoding="UTF-8"?>
        <mdfeProc xmlns="http://www.portalfiscal.inf.br/mdfe" versao="3.00">
        .
        .
        .
    XML
```

Aplicar nos demais se houver.